src/test/resources/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles/wgs_calling_regions.hg38.chr20.100k.interval_list

```diff
@@ -3368,3 +3368,4 @@
 @PG	ID:1	CL:picard.util.IntervalListTools INPUT=[HG38excludeNs.interval_list, genome.interval_list] OUTPUT=wgs_calling_regions.v3.interval_list SORT=true ACTION=INTERSECT    PADDING=0 UNIQUE=false SCATTER_COUNT=1 INCLUDE_FILTERED=false BREAK_BANDS_AT_MULTIPLES_OF=0 SUBDIVISION_MODE=INTERVAL_SUBDIVISION INVERT=false VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json	PN:IntervalListTools
 chr20	60001	63215	+	. intersection ACGTmer
 chr20	63841	100000	+	. intersection ACGTmer
+chr20	100001	100020	+	. intersection ACGTmer
```

Comment:
I'm adding an extra interval in the test to have the interval list extend beyond the intervals in the input VCF. That is a case that will trigger the `writeMissingIntervals` code.